tests/EventListener/ExceptionListenerTest.php

```diff
@@ -52,14 +53,16 @@ public function testHandleException(): void
         self::assertTrue($response->headers->has('Access-Control-Allow-Headers'));
         self::assertEquals(500, $response->getStatusCode());
         $responseBody = json_decode((string) $response->getContent(), true);
-        self::assertCount(4, $responseBody);
+        self::assertCount(5, $responseBody);
         self::assertStringStartsWith('exception-', $responseBody['exceptionId']);
         self::assertEquals('12', $responseBody['code']);
         self::assertEquals('Internal Server Error occurred.', $responseBody['error']);
         self::assertEquals('error', $responseBody['status']);
+        self::assertEquals(null, $responseBody['context']);
         $record = $handler->getRecords()[0];
         self::assertEquals(Logger::CRITICAL, $record['level']);
         self::assertEquals($exception, $record['context']['exception']);
+        self::assertEquals(null, $responseBody['context']['context']);
```

Comment:
```suggestion
        self::assertEquals(null,$$record['context']['context']);
```

Reply:
kůl na 7.4 to aj failnulo, o důvod víc proč sa bát upgradu na connection :D

Reply:
jj, ten "Trying to access array offset on value of type null" uz nam parkrat zatopil